native/src/response.js

```diff
@@ -0,0 +1,15 @@
+export function error({ message }) {
+  return responsify({ "status": "error", "errors": [message]});
+}
+
+export function fatal({ message }) {
+  return responsify({ "status": "fatal", "errors": [message]});
+}
+
+export function ok(ast) {
+  return responsify({ast, "status": "ok"});
+}
+
+function responsify(resp) {
+  return resp;
```

Comment:
Ooooooooh okay. I don't think it's gonna change but better check with @smola, yeah